workflows/alevin-quant/trace.txt.1

```diff
@@ -0,0 +1,13 @@
+task_id	hash	native_id	name	status	exit	submit	duration	realtime	%cpu	peak_rss	peak_vmem	rchar	wchar
```

Comment:
I see you updated the trace and HTML files in https://github.com/AlexsLemonade/alsf-scpca/pull/18/commits/437842bf5c04fc3002bacd8a3ea02b4992ea8012 and this file is the "keep old trace for reference" based on the diff. I might suggest we name this something else for future us reasons. I would expect the other file to be the old one just looking at the file names. (This comment extends to the HTML files.)

Reply:
Yeah, this was how `nextflow` handled the file name conflict. I would have expected the same as you did based on file names, but I am hesitant to rename them just for consistency. Scratch that, I will rename with a date.